### PR TITLE
Bookstore: initialize project with components (v1.2)

### PR DESCRIPTION
**Pull request by mrjuda**
- The building blocks of the app should be set as re-usable components.
- Create a directory for your components.
- Add React Router and set two <Route>s and <Link>s for the app's navigation: Books and Categories.
- Books - the default view: Should display the list of books (empty at this point but it should be ready for the data) with the Remove button (no functionality yet).
- Create a component called Book for displaying a single book (receive a title and an author as a prop) and reuse it in a component that displays a list of books.
- Add a form for adding a book (no functionality yet).
- Create a separate component for this form.
- Categories: Should display a button "Check status" only.

### CODE REVIEW

File: src/components/pages/WebHeader.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    <div className="WebHeader">
+      <h1 className="HeaderTitle">
+        Bookstore
+      </h1>
+      <ul>
+        <li>|</li>
+        <li><a href="/">Books</a></li>
+        <li>|</li>
+        <li><a href="/WebCategories">Categories</a></li>
+        <li>|</li>
```
Comment (bimbolabuari) [inline src/components/pages/WebHeader.js]: As required, Let's use `<Link>`  in the navbar which leads to the booklist and categories.
> image

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hi @mrjuda, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- The project is initialized using `create-react-app`
- Has  `<Route>` for the BookList component and the Categories component

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/pages/WebHeader.js (hunk 1 above)
Comment (mrjuda) [reply src/components/pages/WebHeader.js]: Done, thank you!

**Review by JuliCarracedo — APPROVED** [review]
# Status: Approved✔️
Hi @mrjuda , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**